getting-started/coaddition.rst

```diff
@@ -117,7 +117,7 @@ Coadding warped images
 
 Now you'll assemble the warped images into coadditions for each patch with the ``assembleCoadd`` pipeline.
 As before, we will run without a data query to process a subset of the data, but a selection can be made with the ``-d`` argument just as with warping.
-In this case the the ``-d`` arguement could be omitted since the coaddition process will only find the warps from the previous command and will thus only produce coadds for those patches.
+In this case the the ``-d`` argument could be omitted since the coaddition process will only find the warps from the previous command and will thus only produce coadds for those patches.
```

Comment:
While we're in here... The use of "warp" as a noun is clearly jargon.  At the start of this file "warp" is sort of defined, as a verb: "... warp (retroject) ..." (currently line 17).  Could we use quotation marks there, at the point of first definition (for consistency with the other changes we've made) and then here (on line 120) just say "... find the warped images from ..."?

Finally, I haven't read the whole tutorial thoroughly, but I'm wondering whether there's any place where we're giving people the mapping between the DatasetType strings and the English/jargon terms that we're using for the different intermediate and final products discussed in this documentation.

Reply:
There's a brief mention of `calexp` and `raw` in part 3 (Display) but I think the tutorial would be a lot clearer with more said about this.  It would make Figure 1 in part 2 (Single Frame) easier to understand, I think, too.

Reply:
The detailed edits suggested in this conversation have now been implemented in [4956737](https://github.com/lsst/pipelines_lsst_io/pull/292/commits/4956737533bc7df8497e373e996353c14da3b230).

Reply:
I have now added an explanation/introduction of the notion of `DatasetType` in section 1 of the getting started tutorial ([1a99f3b](https://github.com/lsst/pipelines_lsst_io/pull/292/commits/1a99f3b62c68a7d7912f110473c244c9043e77a7)). I note that, although the concept of a "warp" is mentioned variously throughout this getting started tutorial, I did not find any actual mention of a specific ``DatasetType`` associated with the warps produced (I believe there are multiple different ``DatasetTypes`` that could correspond to warps).